lib/ex_sdp/attribute/fmtp.ex

```diff
@@ -181,6 +204,9 @@ defimpl String.Chars, for: ExSDP.Attribute.FMTP do
         Serializer.maybe_serialize("max-br", fmtp.max_br),
         Serializer.maybe_serialize("level-asymmetry-allowed", fmtp.level_asymmetry_allowed),
         Serializer.maybe_serialize("packetization-mode", fmtp.packetization_mode),
+        Serializer.maybe_serialize("apt", fmtp.apt),
+        Serializer.maybe_serialize("repair-window", fmtp.repair_window),
+        Serializer.maybe_serialize("redundancy-range", fmtp.redundancy_range),
```

Comment:
the logic for serializing the tuple to a "from-to" string seems to be missing

Reply:
After checking once again situation with redundancy-range isn't occur even with simulcast, so I will remove this param